src/controllers/midi/midicontroller.cpp

```diff
@@ -292,46 +292,52 @@ void MidiController::processInputMapping(const MidiInputMapping& mapping,
             return;
         }
 
-        std::visit(MidiUtils::overloaded{
-                           [this, pEngine, channel, control, value, status](
-                                   const ConfigKey& configKey) {
-                               QJSValue function = pEngine->wrapFunctionCode(configKey.item, 5);
-                               const auto args = QJSValueList{
-                                       channel,
-                                       control,
-                                       value,
-                                       status,
-                                       configKey.group,
-                               };
-                               if (!pEngine->executeFunction(function, args)) {
-                                   qCWarning(m_logBase) << "MidiController: Invalid script function"
-                                                        << configKey.item;
-                               }
-                               return;
-                           },
-                           [this, pEngine, channel, control, value, status](
-                                   const QJSValue& function) {
-                               const auto args = QJSValueList{
-                                       channel,
-                                       control,
-                                       value,
-                                       status,
-                               };
-                               if (!pEngine->executeFunction(function, args)) {
-                                   qCWarning(m_logBase)
-                                           << "MidiController: Invalid "
-                                              "anonymous script function";
-                               }
-                               return;
-                           }},
+        return std::visit(
+                MidiUtils::overloaded{
```

Comment:
so now this is causing build failures on ubuntu... I guess try removing it again, maybe its not necessary anymore now that we target `main` instead of 2.4?